catalog/models.py

```diff
@@ -17,7 +43,7 @@ def get_public_components():
     """
     Returns only the components for which publicAccess is true
     """
-    return [c for c in get_components() if c['publicAccess']]
+    return [c for c in get_components_neo4j() if c['publicAccess']]
```

Comment:
Let's add the option for this to be either/or. Either a new config, such as `neo4j_backend`. If True, we use neo4j, otherwise we run with whatever is packaged. That, or if `neo4j_pw` and `neo4j_user` then use the neo4j functions.

It might make things trickier to work with locally if we don't do this.